adjacent_motif_feature_generator.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+import itertools
+import logging as log
+
+from common import mutate_string, NUCLEOTIDES
+from generic_feature_generator import GenericFeatureGenerator
+
+class AdjacentMotifFeatureGenerator(GenericFeatureGenerator):
+    """
```

Comment:
can you describe this in more detail?

i think it is a k-mer motif feature generator for when the motif is some distance away.